geode-junit/src/main/java/org/apache/geode/test/junit/rules/gfsh/GfshRule.java

```diff
@@ -199,4 +220,23 @@ private void stopMembers(GfshExecution gfshExecution) {
     }
     execute(GfshScript.of(stopMemberScripts).withName("Stop-Members"));
   }
+
+  public static String startServerCommand(String name, int port, int connectedLocatorPort) {
```

Comment:
These new static methods don't really fit well in GfshRule. I think you should probably move them to two new classes (probably not a rule) named something like ServerCommandBuilder and LocatorCommandBuilder, and explode the parameters to each have their own setter type method:
```
String startServerCommand = new ServerCommandBuilder()
    .withPort(port)
    .withLocator(locatorPort)
    .create(name);
```

Reply:
There are actually a couple existing classes like this that we could move to geode-dunit and make any necessary changes:
```
geode-core/src/integrationTest/java/org/apache/geode/distributed/LocatorCommand.java
geode-core/src/integrationTest/java/org/apache/geode/distributed/ServerCommand.java
```

Reply:
We could commit what you have ready and then make further changes. I'll go ahead and approve and let you decide.

Reply:
Thanks! I will merge this now and we can make improvements on it when we get to consolidating all these methods into one place.